TicTacToe.Console/ConsoleWrapper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+namespace TicTacToe.ConsoleGame
+{
+    public class ConsoleWrapper : IConsole
+    {
+        public void Write(string message)
+        {
+            Console.WriteLine(message);
+        }
+
+        public void WriteArray(char[] array)
+        {
+            int x = 0;
+            for(int i=0; i<3; i++)
+            {
+                for (int j = 0; j < 3; j++)
+                {
+                    Console.Write(array[x]);
+                    x++;
+                }
+                Console.WriteLine();
+            }
+        }
+
+        public void WriteBool(bool message)
```

Comment:
Do you need to display a boolean in a TicTacToe game?

Reply:
Hi Hsaun, Thanks for the observation. I was trying out an implementation with Emma on that last week, but I won't be making use of it as I have been focussing more on getting the game business logic right. After I get a go-ahead form Rugamba to start working on the console, I will refactor that and implement the logic I have to work on the console.